### DISCUSSION TITLE
Does mi band 7 supports zepp os?

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#63] [Q&A] [answered]
I was just wondering about it because I saw so many people installing apps on it. So is it possible to install apps on mi band 7. And I wanted the code of water drinking reminders app please and BMI too please

**Comment by htoooth** [comment]
Hi @alexmathew11 :
Xiaomi Mi Band 7 is zeppos 1.0. But it does not support installing apps, only watchface. 
Amazfit GT4  series support for apps and watchface.

**Reply by alexmathew11** [reply]
But actually my friends try to install it and it work, it works just as good as Amazfit band 7.

**Reply by OnDriveLine** [reply]
because xiaomi said it no support,so huami said not support,actually it support.

**Reply by alexmathew11** [reply]
What are you trying to say?

**Comment by leober-ramos33** [comment, marked as answer]
Yes, if it works. I have several mini apps that I found on the site: https://amazfitwatchfaces.com/ and also on this forum: https://www.bandbbs.cn/.

I installed a calculator, a calendar (which I changed because it came out in Chinese), and two games. You can find some of the ones I tell you about in this repository: https://github.com/melianmiko/smartband7-apps

And yes, the Smart Band 7 comes with Zepp OS.

The one that I found most interesting is being able to generate 2FA codes on the Smart Band, so when you don't have your mobile phone, you can generate this code from the watch itself, for example, to log in to Google from an unknown device.

But I have not been able to download it from the forum, if someone can download it and send it to me it would be of great help, this is the link: https://www.bandbbs.cn/threads/4955

**Reply by juan518munoz** [reply]
There's a repository for a 2FA app on GitHub
https://github.com/ZoLArk173/Authenticator

**Reply by leober-ramos33** [reply]
Oh thank you very much. The code will help me to build my own to learn. I can also download BandTOTP and it works wonderfully, really very good.

It works for me when I'm at university and I have to log in and I don't bring my phone (because of insecurity), and the Mi band 7 looks more like a random bracelet and thieves are looking for a phone more than a bracelet.
